spec/models/medical_history_spec.rb

```diff
@@ -18,7 +18,7 @@
     it_behaves_like 'a record that is deletable'
   end
 
-  describe '#risk_history?' do
+  describe '#indicates_risk?' do
```

Comment:
If we remove the `indicates_risk?` method, we should also get rid of these tests.